### DISCUSSION TITLE
Opschalingsanalyse vraag 1: Wat wijzigt er in de werkwijze van de gemeente door het sensorenregister?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#251] [Pilot Gemeente]
VNG Opschalingsanalyse vraag 1: **Wat wijzigt er in de werkwijze van de gemeente door het sensorenregister?**

En aanvullend: 
- Welke werkprocessen bij gemeenten zijn nodig voor het registreren van de sensoren?
- Welke (werk)processen bij gemeente zijn nodig voor het handhaven van eventuele verplichtingen?

**Comment by WOostdam** [comment]
Allereerst het centraal registreren van de eigen sensoren. Dat betekent dat bij aanschaf, verwijdering en verandering van sensoren deze info geregistreerd moet worden.

**Reply by marcvanandel** [reply]
'Centraal' als in 'centraal binnen de gemeente' ... en dat ook nog eens landelijk centraal (hopelijk op gemakkelijke en goedkope wijze), right?

**Comment by Marcel-Pnin** [comment]
Maken beleid
Kiezen (of volgen landelijk beleid) wat scope van register is (sensortypen, locaties, observatiedoelen, aantallen, verplicht, vrijwillig, ...) en inregelen evt. juridische voorwaarden (APV)

Afspreken en beheren afspraken
Maken afspraken over aan te leveren info en werkwijze van aanlevering (contract, overeenkomst, SLA(intern), APV, vrijwillig registratieprotocol, ...)

Communiceren met doelgroepen

Valideren/Beoordelen
Validatie (volgens afspraak) van ontvangen afgesproken sensor-informatie. Bepalen wat bij goed- en afkeuren moet gebeuren (bv. opnieuw laten aanleveren of bv. labelen van info of besluit sturen en/of signaal naar handhaving).

Registreren al aanwezige info in SensrNet (registreren per stuk of meer sensoren tegelijkertijd)
Initeel: per sensortype eenmalig verzamelen info, completeren info en registreren alle aanwezige sensoren bij de gemeente zelf
Structureel: registreren per stuk of registreren meer sensoren tegelijkertijd als het 'beter' is als de info standaard uit beheersystemen of GIS/BI-tools komt of als info via bestanden worden aangeleverd.

Bewaken kwaliteit van geregistreerde informatie.
Met name periodiek navragen actualiteit. Bepalen wat bij diverse reacties moet gebeuren. 

Beheren proces
Monitoren voortgang, acteren op lokale en landelijke signalen

Handhaven

**Reply by Marcel-Pnin** [reply]
Marcel van Roij - Tilburg

**Reply by mikegroels** [reply]
Handhaven is volledig afhankelijk van:
- wettelijke verplichting (eventueel in de toekomst)
- welke sensoren willen / moeten we registreren (indien geen wet, maar gemeentelijke verordening / beleid)
- insteek landelijke instanties (zoals RIVM, Rijkswaterstaat, etc.)
Dit is allemaal nog niet duidelijk, dus er kan nog weinig zinnigs over gezegd worden.

**Comment by marcvanandel** [comment]
Interne afspraken en dienstverlening. ISO 19650 geeft hier al een goede invulling / voorstel voor (vanuit asset management). Tip van Tilburg
